### PR TITLE
Move to paket, add linting

### PR DESCRIPTION
**Pull request by Shmew**

### CODE REVIEW

File: Feliz.sln [hunk 1]
```diff
@@ -19,24 +19,11 @@ Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Feliz.Popover", "Feliz.Popo
 EndProject
 Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Tests", "tests\Tests.fsproj", "{74AC6D90-9BF7-44B7-ACDD-6C7616CE07E7}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{2979FCC6-85A7-4C95-B05B-D9A5F48B3C6A}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "web", "web", "{2979FCC6-85A7-4C95-B05B-D9A5F48B3C6A}"
```
Comment (l3m) [inline Feliz.sln:22]: I prefer the original version with 1 folder with these files...

**Comment by Zaid-Ajaj** [comment]
Any ideas why the build is failing? I am guessing it could be due to FAKE version mismatch

**Comment by Shmew** [comment]
There was a wonky merge issue that was fixed. The only problem now is the puppeteer tests are failing due to timeout... it works fine with `npm run test:live`

**Comment by Zaid-Ajaj** [comment]
I want to revisit this soon. I am not a big fan of the fact that we are using nuget and paket at the same time and their config is kinda fighting with each other. I want to simplify all these things and use only one package management system: **nuget**. I know many people are fans of paket but to me it complicates things more than it solves. I also want to move the `build.fsx` file into a console application (that uses nuget) following what I am doing with [Snowflaqe](https://github.com/Zaid-Ajaj/Snowflaqe) which makes things really simple

**Comment by Shmew** [comment]
Yeah sure thing, I didn't delete the branch so you can just clone it if/when you want to mess with it. It was just getting so out of date I figured I should go ahead and close it for now.

**Pull request closed without merging**